build.assets/Dockerfile-centos7

```diff
@@ -247,7 +285,9 @@ COPY --from=libpcsclite \
     /usr/local/lib/
 
 # Download pre-built CentOS 7 assets with clang needed to build BPF tools.
-RUN cd / && curl -L https://s3.amazonaws.com/clientbuilds.gravitational.io/go/centos7-assets.tar.gz | tar -xz
+#RUN cd / && curl -L https://s3.amazonaws.com/clientbuilds.gravitational.io/go/centos7-assets.tar.gz | tar -xz
```

Comment:
Yesssss. Get rid of this with all haste, please. We pay a lot of $$$ every month for this thing getting downloaded constantly.

Reply:
@reedloden I think we can keep the "centos assets" as a container in GitHub. I just need to coordinate this with someone.